go/src/lib.rs

```diff
@@ -35,22 +35,27 @@ pub struct CommandResponse {
     float_value: c_double,
     bool_value: bool,
 
```

Comment:
This struct urgently needs a type parameter

Reply:
Yes, I agree. The next PR will address this.